android/pytorch-native/README.md

```diff
@@ -1,54 +1,64 @@
 
 # Pytorch Android Build Procedure
 
+Follow this setup guide in order to run DJL apps on an Android. In order to successfully run the apps, you must install several Android items and build Android ABIs outlined below.
```

Comment:
We should make it clear this is not for regular user, this is only for developers who want to build pytorch naitve binaries from source.